src/Illuminate/Console/GeneratorCommand.php

```diff
@@ -202,9 +203,9 @@ protected function qualifyModel(string $model)
             return $model;
         }
 
-        return is_dir(app_path('Models'))
-                    ? $rootNamespace.'Models\\'.$model
-                    : $rootNamespace.$model;
+        return class_exists($rootNamespace.$model)
+            ? $rootNamespace.$model
+            : $rootNamespace.'Models\\'.$model;
```

Comment:
This entire change doesn't make sense I don't think. For most generator commands it's going to cause unexpected behavior.